testing/catalog.go

```diff
@@ -130,6 +144,24 @@ spec:
 `)
 }
 
+// EiriniStagingAppYaml returns a fake Eirini staging app yaml
```

Comment:
maybe add `// it's a staging app because it doesn't have the Eirini label` (maybe include the actual label in the comment?)